lib/prometheus/business_collector.rb

```diff
@@ -84,6 +101,50 @@ def collect
     @client_policies.observe(
       Policy.where(account_id: client_accounts.select(:id)).count
     )
+
+    sq = Policy.joins(:profiles).select(:account_id, "profiles.ref_id").distinct
+    Policy.select('sq.ref_id', 'COUNT(sq.account_id)')
+          .from("(#{sq.to_sql}) as sq")
+          .group('sq.ref_id').each do |item|
```

Comment:
I guess the subquery in a from clause is to ensure distinct counting. Postgresql & Rails supports `COUNT(DISTINCT ...)`.
```
> Policy.joins(:profiles).select(:account_id, "profiles.ref_id").distinct.group(:ref_id).count(:account_id)
   (4.7ms)  SELECT COUNT(DISTINCT "policies"."account_id") AS count_account_id, "ref_id" AS ref_id FROM "policies" INNER JOIN "profiles" ON "profiles"."policy_id" = "policies"."id" GROUP BY "ref_id"
```